### PR TITLE
new item for pirates: hand cannon

### PR DESCRIPTION
**Pull request by LazennG**
# Document the changes in your pull request
handcannon
Gives the pirate ship a new weapon called the hand cannon which can instantly fuck up doors and windows, but only gets 3 shots. 
# Wiki Documentation
(https://wiki.yogstation.net/wiki/Pirate)
in the ship section, add something along the lines of
"In the bridge is the breaching weapon: hand cannon, along with 3 cannonballs.
:cl:  
rscadd: Added new pirate weapon: hand cannon
/:cl:
requested from and sprited by @cuackles

### CODE REVIEW

File: code/game/objects/items/pneumaticCannon.dm [hunk 1]
```diff
@@ -339,3 +339,8 @@
 /obj/item/storage/backpack/magspear_quiver/PopulateContents()
 	for(var/i in 1 to 20)
 		new /obj/item/throwing_star/magspear(src)
+
+
```
Comment (ToasterBiome) [inline code/game/objects/items/pneumaticCannon.dm]: no

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: code/game/objects/items/pneumaticCannon.dm (hunk 1 above)
Comment (tattax) [reply code/game/objects/items/pneumaticCannon.dm]: lol

Comment (LazennG) [reply code/game/objects/items/pneumaticCannon.dm]: oops forgot to clean this up from when i tried making this thing a pneumatic cannon in the first place

**Review by nesisknesis — APPROVED** [review]

File: code/modules/projectiles/boxes_magazines/internal/grenade.dm [hunk 2]
```diff
@@ -15,3 +15,9 @@
 	ammo_type = /obj/item/ammo_casing/caseless/rocket
 	caliber = "84mm"
 	max_ammo = 1
+
+/obj/item/ammo_box/magazine/internal/cannonball
+	name = "cannonball"
+	ammo_type = /obj/item/ammo_casing/caseless/cannonball
+	caliber = "30mm"
```
Comment (ToasterBiome) [inline code/modules/projectiles/boxes_magazines/internal/grenade.dm]: Cannonballs smaller than 1.5 inches were grapeshot and were more like a shotgun, 30mm is 1.1~ inches. should be closer to 100mm (4.1 inches)

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

**Comment by ToasterBiome** [comment]
@LazennG conflicting

**Review by ToasterBiome — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Comment by 00aquir** [comment]
bug https://github.com/yogstation13/Yogstation/issues/13621

**Pull request merged**